src/Bundle/CoreBundle/Tests/Service/DomainDefinitionParserTest.php

```diff
@@ -15,60 +15,255 @@
+                    "title": "Other Content Type",
+                    "identifier": "other_ct",
+                    "icon": "file",
+                    "content_label": "{type} #{id}",
+                    "fields": [],
+                    "views": [
+                        {
+                            "title": "All",
+                            "identifier": "all",
+                            "type": "table"
+                        }
+                    ],
+                    "permissions": {
+                        "view content": "true",
+                        "list content": "true",
+                        "create content": "member.type == \"editor\"",
+                        "update content": "member.type == \"editor\"",
+                        "delete content": "member.type == \"editor\""
+                    }
+                }
+            ],
+            "setting_types": [
+                {
+                    "title": "First Setting Type",
+                    "identifier": "first_st",
+                    "icon": "file",
+                    "fields": [
+                        {
+                            "title": "Title",
+                            "identifier": "title",
+                            "type": "text"
+                        }
+                    ],
+                    "permissions": {
+                        "view setting": "true",
+                        "update setting": "false"
+                    },
+                    "webhooks": [
+                        {
+                          "query": "query { type }",
+                          "url": "https://example.com",
+                          "check_ssl": true,
+                          "content_type": "json",
+                          "condition": "true"
+                        }
+                    ]
+                },
+                {
+                    "title": "Other Setting Type",
+                    "identifier": "other_st",
+                    "fields": [],
+                    "permissions": {
+                        "view setting": "true",
+                        "update setting": "member.type == \"editor\""
+                    }
+                }
+            ],
+            "domain_member_types": [
+                {
+                    "title": "Any",
+                    "identifier": "any",
+                    "domain_member_label": "Any",
+                    "fields": [
+                        {
+                            "title": "Title",
+                            "identifier": "title",
+                            "type": "text"
+                        }
+                    ]
+                }
+            ],
+            "permissions": {
+                "view domain": "true",
+                "update domain": "true"
             }
-        ],
-        "permissions": {
-            "view domain": "true",
-            "update domain": "false"
-        }
-    }';
+        }';
+    }
 
```

Comment:
As writing files is dangerous, i would do one additional test which checks if an path with invalid chars (fe test/../bala) can be written.
I don't know why we should but maybe we will change the identifier regex patterns some day.

Reply:
implemented a reverse test